PaddleCV/video/train.py

```diff
@@ -140,9 +140,6 @@ def train(args):
             optimizer.minimize(train_loss)
             train_pyreader = train_model.pyreader()
 
-    if not args.no_memory_optimize:
```

Comment:
maybe should remove args: no_memory_optimize

Reply:
yes, if this can be confirmed no need in future codes.